roles/ceph-rgw-loadbalancer/defaults/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+# Variables here are applicable to all host groups NOT roles
+
+# This sample file generated by generate_group_vars_sample.sh
```

Comment:
this comment shouldn't be there

Reply:
ok, will remove it.

Reply:
do not forget to run the `./generate_group_vars_sample.sh` script

thanks!

Reply:
oh, that's a missing point, I shall use this facility.